Assets/Scripts/Interface/Layers/LayerToggleBar.cs

```diff
@@ -69,7 +69,7 @@ void OnLayerVisibilityLockChanged(AbstractLayer a_layer, bool a_locked)
 		void OnInfoButtonClicked()
 		{
 			string mediaUrl = MediaUrl.Parse(m_layer.Media);
-			InterfaceCanvas.Instance.webViewWindow.CreateWebViewWindow(mediaUrl);
+			MediaUrl.Parse(mediaUrl);
```

Comment:
Basing just on what I saw in the modification in ActivePlanLayerType.cs, shouldn't this be
`Application.OpenURL(mediaURL)`  ?